lib/database/leaderboard.php

```diff
@@ -400,19 +400,21 @@ function GetLeaderboardEntriesDataJSON($lbID, $user, $numToFetch, $offset, $frie
     return $retVal;
 }
 
-function GetLeaderboardData($lbID, $user, $numToFetch, $offset, $friendsOnly)
+function GetLeaderboardData($lbID, $user, $numToFetch, $offset, $friendsOnly, $nearby = 0)
```

Comment:
my comment about making `n` be 1 or 0 was purely at the URL parameter level. it would probably make more sense to have this be true/false in the implementation.

Reply:
Sorry that was my bad misinterpreting your meaning, I'll clean this up